x-pack/metricbeat/module/enterprisesearch/health/_meta/docs.asciidoc

```diff
@@ -0,0 +1,3 @@
+This is the health metricset of the module enterprisesearch.
```

Comment:
I think this didn't make it to the actual "docs" section above `metricbeat/docs/modules/enterprisesearch/health.asciidoc`

Reply:
I be honest, I have no idea what I need to do to get that updated. Spent an hour today running all kinds of `mage` commands all over the project and can't seem to get it updated :-(

Reply:
I know the feeling. Try a last `make docs` or `mage docs` on `/metricbeat` (the one on the root, not the one at xpack).

I know some stuff on xpack only works if you run it on OSS folder. It's trivia knowledge and it's a pain. We know it, you are not alone on this :smile: If not don't worry, I'll try to add the docs in a followup PR

Reply:
make docs succeeds, but does not make any changes in the repo :-(

Reply:
Bah, I'll take a look at it in a follow up. No prob